src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlElasticPoolActivity.md

```diff
@@ -89,6 +89,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -PassThru
+{{Fill PassThru Description}}
```

Comment:
@jugeorge please make this description "Specifying this will return output if the command was successful."